### ISSUE TITLE
Expanded Squiz.Arrays.ArrayDeclaration.SpaceBeforeComma support

### ISSUE THREAD

**Issue by earthiverse** [PHPCSStandards/PHP_CodeSniffer#477]
## Is your feature request related to a problem?
Right now, the following code is correctly sniffed with `Squiz.Arrays.ArrayDeclaration.SpaceBeforeComma`
```php
$arr1 = [
    'a' => 1,
    'b' => 2
    , // Correctly errors with Squiz.Arrays.ArrayDeclaration.SpaceBeforeComma
    'c' => 3,
];
```

However, if you have different "length" keys (in this example, `aa` is two characters), it no longer warns
```php
$arr1 = [
    'a' => 1,
    'b' => 2
    , // No error
    'aa' => 3,
];
```

## Describe the solution you'd like
I would like the ability to enable this sniff even if keys are of different "lengths"

## Additional context (optional)

- [ ] I intend to create a pull request to implement this feature.

**Comment by jrfnl** [comment]
@earthiverse The issue you have identified is due to a [long-time known design-flaw](https://github.com/squizlabs/PHP_CodeSniffer/issues/582) (one error hiding behind another) in the sniff and will not be fixed. Instead, new, more configurable sniffs will be made available to eventually replace the `Squiz.Arrays.ArrayDeclaration` sniff.

In the mean time, I'd like to suggest the [`Universal.WhiteSpace.CommaSpacing`](https://github.com/phpcsstandards/phpcsextra#universalwhitespacecommaspacing-wrench-bar_chart-books) sniff as a way to still check the spacing around the comma, even when you have the `Squiz.Arrays.ArrayDeclaration.DoubleArrowNotAligned` eror silenced.

**Comment by earthiverse** [comment]
[Universal.WhiteSpace.CommaSpacing](https://github.com/phpcsstandards/phpcsextra#universalwhitespacecommaspacing-wrench-bar_chart-books) works, thanks!
